sipa/config/default.py

```diff
@@ -70,3 +70,5 @@
     "Strehlener Platz",
     "Weberplatz"
 ]
+
+SESSION_REFRESH_EACH_REQUEST = False
```

Comment:
@sgeisler Please add according documentation as being done in the sample config file.